solidity/test/helpers/Constants.js

```diff
@@ -1,27 +1,31 @@
-const { web3 } = require('@openzeppelin/test-environment');
+const { BigNumber, ethers } = require('ethers');
 
 module.exports = {
     registry: {
-        CONTRACT_REGISTRY: web3.utils.asciiToHex('ContractRegistry'),
-        BANCOR_NETWORK: web3.utils.asciiToHex('BancorNetwork'),
-        NETWORK_SETTINGS: web3.utils.asciiToHex('NetworkSettings'),
-        CONVERTER_FACTORY: web3.utils.asciiToHex('ConverterFactory'),
-        CONVERSION_PATH_FINDER: web3.utils.asciiToHex('ConversionPathFinder'),
-        CONVERTER_UPGRADER: web3.utils.asciiToHex('BancorConverterUpgrader'),
-        CONVERTER_REGISTRY: web3.utils.asciiToHex('BancorConverterRegistry'),
-        CONVERTER_REGISTRY_DATA: web3.utils.asciiToHex('BancorConverterRegistryData'),
-        BNT_TOKEN: web3.utils.asciiToHex('BNTToken'),
-        BANCOR_X: web3.utils.asciiToHex('BancorX'),
-        BANCOR_X_UPGRADER: web3.utils.asciiToHex('BancorXUpgrader'),
-        NETWORK_SETTINGS: web3.utils.asciiToHex('NetworkSettings')
+        CONTRACT_REGISTRY: ethers.utils.formatBytes32String('ContractRegistry'),
+        BANCOR_NETWORK: ethers.utils.formatBytes32String('BancorNetwork'),
+        NETWORK_SETTINGS: ethers.utils.formatBytes32String('NetworkSettings'),
+        CONVERTER_FACTORY: ethers.utils.formatBytes32String('ConverterFactory'),
+        CONVERSION_PATH_FINDER: ethers.utils.formatBytes32String('ConversionPathFinder'),
+        CONVERTER_UPGRADER: ethers.utils.formatBytes32String('BancorConverterUpgrader'),
+        CONVERTER_REGISTRY: ethers.utils.formatBytes32String('BancorConverterRegistry'),
+        CONVERTER_REGISTRY_DATA: ethers.utils.formatBytes32String('BancorConverterRegistryData'),
+        BNT_TOKEN: ethers.utils.formatBytes32String('BNTToken'),
+        BANCOR_X: ethers.utils.formatBytes32String('BancorX'),
+        BANCOR_X_UPGRADER: ethers.utils.formatBytes32String('BancorXUpgrader'),
+
+        //
```

Comment:
If you think that an explicit comment break is needed here, then let's add an explicit description ("this is needed for legacy tests which need to access...")